### PR TITLE
[jdbc] Add mssql support for scan operation

### PR DESCRIPTION
**Pull request by KowsarAtz**
Fixing the syntax issue SQL Server faced during scan operation.
( Microsoft SQL Server doesn't support LIMIT statement, so I added a condition to use TOP instead when we are using jdbc-binding to benchmark SQL Server scan operation. )

### CODE REVIEW

File: jdbc/src/main/java/com/yahoo/ycsb/db/flavors/DBFlavor.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2016 YCSB contributors. All rights reserved.
+ * Copyright (c) 2019 YCSB contributors. All rights reserved.
```
Comment (busbey) [inline jdbc/src/main/java/com/yahoo/ycsb/db/flavors/DBFlavor.java]: This should be 2016, 2019 to reflect work from both periods.

File: jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2010 - 2016 Yahoo! Inc., 2016 YCSB contributors. All rights reserved.
+ * Copyright (c) 2010 - 2019 Yahoo! Inc., 2019 YCSB contributors. All rights reserved.
```
Comment (busbey) [inline jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java]: This should remain 2010-2016 Yahoo and then 2016, 2019 ycsb contributors

File: jdbc/src/main/java/com/yahoo/ycsb/db/flavors/DefaultDBFlavor.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2016 YCSB contributors. All rights reserved.
+ * Copyright (c) 2019 YCSB contributors. All rights reserved.
```
Comment (busbey) [inline jdbc/src/main/java/com/yahoo/ycsb/db/flavors/DefaultDBFlavor.java]: This should be 2016, 2019 to reflect work from both periods.

Comment (KowsarAtz) [reply jdbc/src/main/java/com/yahoo/ycsb/db/flavors/DefaultDBFlavor.java]: Thank you for your review! I've made the changes as requested.

**Pull request merged**